unittests/parser_test_case.py

```diff
@@ -84,3 +87,10 @@ def _test_calldef_exceptions(self, calldef, exceptions):
             (calldef.name,
              pprint.pformat([delc.name for delc in exception_decls]),
              pprint.pformat([delc.name for delc in exceptions_indeed])))
+
+if sys.version_info < (2, 7, 0) and not found_unittest2:
```

Comment:
I know this kind of situation were you have some legacy libraries (very often at work), so I fully understand.

As this is just for the testing part, it will not affect pygccxml itself. I have no strong opinion so lets be flexible. Please remove unitest2 completely and apply the patch for python < 2.7. That'll do.